lib/elixir/test/elixir/map_set_test.exs

```diff
@@ -138,6 +138,10 @@ defmodule MapSetTest do
     refute MapSet.subset?(map_set_v1(7..12), MapSet.new(1..10))
   end
 
+  test "inspect" do
+    assert inspect(MapSet.new([?a])) == "#MapSet<[#{?a}]>"
```

Comment:
wdyt about being very explicit here? This mimics `lib/elixir/test/elixir/inspect_test.exs`.

```suggestion
    assert inspect(MapSet.new([?a])) == "#MapSet<[97]>"
```